lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -1282,29 +1282,6 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     return createRemainderPayload(b, loc, converter, payloadArgs, remTensor,
                                   operands);
   }
-  if (auto fmod = dyn_cast<AtenFmodTensorOp>(op)) {
-    Type newResultType =
-        cast<RankedTensorType>(converter->convertType(fmod.getType()))
-            .getElementType();
-
-    Value self = convertScalarToDtype(b, loc, payloadArgs[0], newResultType);
-    Value other = convertScalarToDtype(b, loc, payloadArgs[1], newResultType);
-    Value result;
-
-    if (isa<mlir::FloatType>(newResultType)) {
-      Value n = b.create<arith::DivFOp>(loc, self, other);
-      n = b.create<math::TruncOp>(loc, n);
-      Value n_y = b.create<arith::MulFOp>(loc, n, other);
-      result = b.create<arith::SubFOp>(loc, self, n_y);
-    } else if (isa<mlir::IntegerType>(newResultType)) {
-      Value n = b.create<arith::DivSIOp>(loc, self, other);
-      Value n_y = b.create<arith::MulIOp>(loc, n, other);
-      result = b.create<arith::SubIOp>(loc, self, n_y);
-    } else {
-      fmod.emitError("Unsupported type encountered for AtenFmodTensorOp.");
-    }
-    return result;
-  }
```

Comment:
You need to remove it from certain other places in the same file. Please search for the op name and remove it.

Reply:
Oops. take two :) Can you take another look?